contracts/payloads/IGP26/PayloadIGP26.sol

```diff
@@ -657,6 +707,22 @@ contract PayloadIGP26 {
         {
             IFluidVaultT1(newVaultAddress).updateRebalancer(0x264786EF916af64a1DB19F513F24a3681734ce92);
         }
+
+        // Approve new vault to spend the reserves dust tokens
+        {
+            address[] memory protocols = new address[](2);
+            address[] memory tokens = new address[](2);
+
+            {   
+                protocols[0] = newVaultAddress;
+                tokens[0] = newConstants.supplyToken;
+
+                protocols[1] = newVaultAddress;
+                tokens[1] = newConstants.borrowToken;
+
+                FLUID_RESERVE.approve(protocols, tokens, amounts);
```

Comment:
if the token is ETH, we need to skip setting any allowance for the token via reserveContract.approve(), as this is not implemented there (yet).